lib/client.js

```diff
@@ -1,20 +1,30 @@
-'use strict';
+'use strict'
 
-var https = require('https');
-var pjson = require('../package.json');
-var Xml2js = require('xml2js');
-var parser = new Xml2js.Parser({explicitArray: false});
+const https = require('https')
+const pjson = require('../package.json')
+const Xml2js = require('xml2js')
 
 exports.create = function (config) {
-  config.RECURLY_HOST = config.SUBDOMAIN + '.recurly.com';
+  config.RECURLY_HOST = config.SUBDOMAIN + '.recurly.com'
+  const xml2jsOptions = config.XML2JS_OPTIONS || null
+  if(xml2jsOptions && Array.isArray(xml2jsOptions.valueProcessors)) {
+    xml2jsOptions.valueProcessors = xml2jsOptions.valueProcessors.map((maybeFunction) => {
+      if(typeof maybeFunction === 'string' && typeof Xml2js.processors[maybeFunction] === 'function') {
+        return Xml2js.processors[maybeFunction]
+      }
+      return maybeFunction
+    })
+  }
+
+  const parser = new Xml2js.Parser(config.XML2JS_OPTIONS || { explicitArray: false })
   return {
 
     request: function (route, callback, data) {
-      var endpoint = route[0];
-      var method = route[1];
-      var headers = route[2];
-      var that = this;
-      var options = {
+      const endpoint = route[0]
```

Comment:
Yeah my thinking was that support is coming to an end for <4.x at the end of the year: https://github.com/nodejs/LTS

Are you cool with bumping to 4.x?